gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/query/visitor/ExecutionVisitor.java

```diff
@@ -170,6 +191,12 @@ public EntityFetcherResponse visit(SelectionNode selectionNode) {
                   EntitiesRequest.newBuilder(executionContext.getEntitiesRequest())
                       .clearSelection()
                       .clearFilter()
+                      // TODO: Should we push order by, limit and offet down to the data source?
+                      // If we want to push the order by down, we would also have to divide order by into
+                      // sourceToOrderBySelectionExpressionMap, sourceToOrderByMetricExpressionMap, sourceToOrderByTimeAggregationMap
```

Comment:
sourceToOrderByTimeAggregationMap probably doesn't make sense. Time aggregations is for fetching time series where order by is not relevant.
But, the other two are valid and I think should be done. If not in this PR a separate one can follow.